Scheduled Updates: Wrap the single site context with the Card component

bogiii opened the pull request:
Closes https://github.com/Automattic/wp-calypso/issues/91285

## Proposed Changes

* Wrap the single site context with the Card component
* Update style to support smaller resolutions

## Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Go to `/plugins/scheduled-updates/{ATOMIC_SLUG}`
* Check if there is the Card parent component for the `list`, `create`, `edit`, `logs`, and `notifications` screens.

List:
<img width="1015" alt="Screenshot 2024-05-31 at 00 00 29" src="https://github.com/Automattic/wp-calypso/assets/1241413/965ab0a4-cfe1-42e5-afc6-555a75615f92">

New Schedule:
<img width="1013" alt="Screenshot 2024-05-31 at 00 00 41" src="https://github.com/Automattic/wp-calypso/assets/1241413/f71bad5b-e0a5-43dd-b564-a69b7ab130aa">

Edit Schedule:
<img width="1002" alt="Screenshot 2024-05-31 at 00 00 53" src="https://github.com/Automattic/wp-calypso/assets/1241413/1ed2b1b9-a2cb-4dcb-95df-9f5fb7e4012b">

Logs:
<img width="1017" alt="Screenshot 2024-05-31 at 00 01 03" src="https://github.com/Automattic/wp-calypso/assets/1241413/201ba7d7-c0ff-47a3-a819-9641dbf22cdd">

Notifications:
<img width="1024" alt="Screenshot 2024-05-31 at 00 01 22" src="https://github.com/Automattic/wp-calypso/assets/1241413/e2149c35-8ef2-43d9-8879-29392fb279b9">


Empty state:
| Multisite | Single |
|--------|--------|
| <img width="1170" alt="Screenshot 2024-05-30 at 23 54 29" src="https://github.com/Automattic/wp-calypso/assets/1241413/8bf109bc-fc33-4284-aa1b-b9f4a63563c9"> | <img width="1728" alt="Screenshot 2024-05-30 at 23 54 47" src="https://github.com/Automattic/wp-calypso/assets/1241413/020bda4e-f359-47b7-8ee6-5cb78231ea83"> |

Smaller resolution:
![Screen Capture on 2024-05-31 at 00-11-10](https://github.com/Automattic/wp-calypso/assets/1241413/6681127a-be8a-4bcf-bfaf-d493946d80ae)

## Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

bogiii committed:
Wrap the create screen with the Card component

bogiii committed:
Add missing main class

bogiii committed:
Wrap the notification settings screen with the Card component

bogiii committed:
Wrap the edit screen with the Card component

bogiii committed:
Wrap the logs screen with the Card component

bogiii committed:
Wrap the list screen with the Card component

bogiii committed:
Add missing margin-bottom style

bogiii committed:
Change breakpoint value

bogiii committed:
Show scrollbar on x-axis if table overflow

bogiii committed:
Center empty state for the single site context

bogiii committed:
Update selector

bogiii committed:
Update table data min width

JanaMW27 commented:
Hello @bogiii, I didn't manage to test this live, but from what I see in the videos, this is perfect!
Thank you very much for the help on this and for being so quick and accommodating.

ouikhuan reviewed (commented):
Nice work @bogiii ! A few observation during my testing (I'm testing using Chrome 125.0.6422.77 ):

The scrollbar is showing by default:
![Screenshot 2024-05-31 at 3 08 28 PM](https://github.com/Automattic/wp-calypso/assets/4074459/185d4e19-1cb8-4a34-9cdb-2c4a02cd0fb4)

On the smaller screen, there's no margin for email notification settings box:
![Screenshot 2024-05-31 at 3 16 01 PM](https://github.com/Automattic/wp-calypso/assets/4074459/6e384385-5635-4817-aafc-2d6d79440f03)

The bottom border is disappeared:

![Screenshot 2024-05-31 at 3 08 51 PM](https://github.com/Automattic/wp-calypso/assets/4074459/31e8b238-3db7-45e0-ad25-2f45b23a3671)

This PR also seems to affected multisite level list too:
![Screenshot 2024-05-31 at 3 25 15 PM](https://github.com/Automattic/wp-calypso/assets/4074459/24d23f1d-1d90-474b-b80b-e7dc81816e0b)

bogiii committed:
Switch overflow scroll to auto

bogiii committed:
Add missing class name

bogiii commented:
> The scrollbar is showing by default:
> This PR also seems to affected multisite level list too:

Fixed. Switched the overflow value from "scroll" to "auto." Now, it should only show if scrolling is necessary.

> On the smaller screen, there's no margin for email notification settings box:
> The bottom border is disappeared:

Fixed, it was a missing class for the main card component.

Hey @ouikhuan, I've resolved the feedback. Could you please take another look?

bogiii committed:
Increase name cell min-width

bogiii committed:
Switch overflow scroll to auto

ouikhuan reviewed (approved):
LGTM!
